pythonEngine.qbs

```diff
@@ -16,6 +17,57 @@ Product {
 
     cpp.cxxLanguageVersion: "c++11"
 
+
+    property string pythonHome: ""
+
+    Probe {
+        id: pythonAndNumpyRecognizer
+        property string app: "python"
+
+        property string pythonRoot
+        property var getPythonRoot: ["-c", "import sys; print(sys.exec_prefix)"]
+
+        property string pythonArch
+        property var getPythonArch: ["-c", "import platform; print(platform.architecture()[0])"]
+
+        property string numpyIncludeDir
+        property var getNumpyIncludeDir: ["-c", "import numpy.distutils; print(numpy.distutils.misc_util.get_numpy_include_dirs())[0]"]
+
+        configure: {
+            var pythonArchBuffer;
+            found = false;
+            var proc = new Process();
+
+            if(proc.exec(app, getPythonRoot) === 0){
+                pythonRoot = proc.readStdOut().trim();
+                found = true;
+            }else{
+                found = false;
+            }
+
+            if(proc.exec(app, getPythonArch) === 0){
+                pythonArchBuffer = proc.readStdOut().trim();
+                if(pythonArchBuffer === "64bit"){
+                    pythonArch = "x86_64"
+                    found = true;
+                }else if(pythonArchBuffer === "32bit"){
+                    pythonArch = "x86"
+                    found = true;
+                }else{
+                    found = false;
+                }
+            }
+
+            if(proc.exec(app, getNumpyIncludeDir) === 0){
+                numpyIncludeDir = proc.readStdOut().trim();
+                found = true;
```

Comment:
Well, imagine you failed to get python root but succeed to get numpy dir, why it should be true?